tests/test_docker.sh

```diff
@@ -7,7 +7,7 @@
 set -ex
 
 # run the model service's tests
-docker-compose -f local.yml run prediction pytest
+docker-compose -f local.yml run prediction pytest -rs
```

Comment:
What does ``-rs`` do? :)

Reply:
It displays a summary of skipped tests :) otherwise, every attempt to print something by a test is captured and only showed if the test fails. [See the doc reference](https://docs.pytest.org/en/latest/example/simple.html)

Reply:
Neat!